src/pepr/operator/crd/sources/cluster-config/v1alpha1.ts

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright 2024 Defense Unicorns
+ * SPDX-License-Identifier: AGPL-3.0-or-later OR LicenseRef-Defense-Unicorns-Commercial
+ */
+
+import { V1CustomResourceDefinitionVersion, V1JSONSchemaProps } from "@kubernetes/client-node";
+
+export const v1alpha1: V1CustomResourceDefinitionVersion = {
+  name: "v1alpha1",
+  served: true,
+  storage: true,
+  schema: {
+    openAPIV3Schema: {
+      type: "object",
+      properties: {
+        spec: {
+          type: "object",
+          properties: {
+            attributes: {
+              type: "object",
+              properties: {
+                clusterName: {
+                  type: "string",
+                  description: "Friendly name to associate with your UDS cluster",
+                },
+                tags: {
+                  type: "array",
+                  description: "Tags to apply to your UDS cluster",
+                  items: {
+                    type: "string",
+                  },
+                },
+              },
+            },
+            networking: {
+              type: "object",
+              properties: {
+                kubeapiCIDR: {
+                  type: "string",
+                  description:
+                    "cidr range for your Kubernetes control plane nodes. This is a manual override that can be used instead of relying on Pepr to automatically watch and update the values",
+                },
+                kubenodeCIDRS: {
+                  type: "array",
+                  description:
+                    "cidr(s) for all Kubernetes nodes (not just control plane). Similar reason to above,annual override instead of relying on watch",
+                  items: {
+                    type: "string",
+                  },
```

Comment:
```suggestion
                kubeApiCIDR: {
                  type: "string",
                  description:
                    "CIDR range for your Kubernetes control plane nodes. This is a manual override that can be used instead of relying on Pepr to automatically watch and update the values",
                },
                kubeNodeCIDRs: {
                  type: "array",
                  description:
                    "CIDR(s) for all Kubernetes nodes (not just control plane). Similar reason to above,annual override instead of relying on watch",
                  items: {
                    type: "string",
                  },
```
Will likely need to regenerate after making these changes, and maybe update some code refs:
1. CIDR is an acronym so should be capitalized everywhere (in the description).
2. CIDRs is just the plural, so the s should be lowercase.
3. To keep with camelCase-ing things `Api` and `Node` should have capital letters in the property keys.